# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Janusz Harkot <jh@blueice.pl>, 2015
# piotrjakimiak <legolass71@gmail.com>, 2015
# Maciek Olko <maciej.olko@gmail.com>, 2015
msgid ""
msgstr ""
"Project-Id-Version: Django REST framework\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-01-30 16:23+0000\n"
"PO-Revision-Date: 2015-03-04 17:03+0000\n"
"Last-Translator: piotrjakimiak <legolass71@gmail.com>\n"
"Language-Team: Polish (http://www.transifex.com/projects/p/django-rest-framework/language/pl/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: pl\n"
"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: authentication.py:69
msgid "Invalid basic header. No credentials provided."
msgstr "Niepoprawny podstawowy nagłówek. Brak danych uwierzytelniających."

#: authentication.py:72
msgid "Invalid basic header. Credentials string should not contain spaces."
msgstr "Niepoprawny podstawowy nagłówek. Ciąg znaków danych uwierzytelniających nie powinien zawierać spacji."

#: authentication.py:78
msgid "Invalid basic header. Credentials not correctly base64 encoded."
msgstr "Niepoprawny podstawowy nagłówek. Niewłaściwe kodowanie base64 danych uwierzytelniających."

#: authentication.py:90
msgid "Invalid username/password."
msgstr "Niepoprawna nazwa użytkownika lub hasło."

#: authentication.py:156
msgid "Invalid token header. No credentials provided."
msgstr "Niepoprawny nagłówek tokena. Brak danych uwierzytelniających."

#: authentication.py:159
msgid "Invalid token header. Token string should not contain spaces."
msgstr "Niepoprawny nagłówek tokena. Token nie może zawierać odstępów."

#: authentication.py:168
msgid "Invalid token."
msgstr "Niepoprawny token."

#: authentication.py:171
msgid "User inactive or deleted."
msgstr "Użytkownik nieaktywny lub usunięty."

#: exceptions.py:38
msgid "A server error occurred."
msgstr "Wystąpił błąd serwera."

#: exceptions.py:73
msgid "Malformed request."
msgstr "Zniekształcone żądanie."

#: exceptions.py:78
msgid "Incorrect authentication credentials."
msgstr "Błędne dane uwierzytelniające."

#: exceptions.py:83
msgid "Authentication credentials were not provided."
msgstr "Nie podano danych uwierzytelniających."

#: exceptions.py:88
msgid "You do not have permission to perform this action."
msgstr "Nie masz uprawnień, by wykonać tę czynność."

#: exceptions.py:93
msgid "Not found."
msgstr "Nie znaleziono."

#: exceptions.py:98
msgid "Method \"{method}\" not allowed."
msgstr "Niedozwolona metoda \"{method}\"."

#: exceptions.py:109
msgid "Could not satisfy the request Accept header."
msgstr "Nie można zaspokoić nagłówka Accept żądania."

#: exceptions.py:121
msgid "Unsupported media type \"{media_type}\" in request."
msgstr "Brak wsparcia dla żądanego typu danych \"{media_type}\"."

#: exceptions.py:134
msgid "Request was throttled."
msgstr "Żądanie zostało zdławione."

#: fields.py:153 relations.py:132 relations.py:156 validators.py:77
#: validators.py:155
msgid "This field is required."
msgstr "To pole jest wymagane."

#: fields.py:154
msgid "This field may not be null."
msgstr "Pole nie może mieć wartości null."

#: fields.py:487 fields.py:515
msgid "\"{input}\" is not a valid boolean."
msgstr "\"{input}\" nie jest poprawną wartością logiczną."

#: fields.py:550
msgid "This field may not be blank."
msgstr "To pole nie może być puste."

#: fields.py:551 fields.py:1324
msgid "Ensure this field has no more than {max_length} characters."
msgstr "Upewnij się, że to pole ma nie więcej niż {max_length} znaków."

#: fields.py:552
msgid "Ensure this field has at least {min_length} characters."
msgstr "Upewnij się, że pole ma co najmniej {min_length} znaków."

#: fields.py:587
msgid "Enter a valid email address."
msgstr "Podaj poprawny adres e-mail."

#: fields.py:604
msgid "This value does not match the required pattern."
msgstr "Ta wartość nie pasuje do wymaganego wzorca."

#: fields.py:615
msgid ""
"Enter a valid \"slug\" consisting of letters, numbers, underscores or "
"hyphens."
msgstr "Wprowadź poprawną wartość pola typu \"slug\", składającą się ze znaków łacińskich, cyfr, podkreślenia lub myślnika."

#: fields.py:627
msgid "Enter a valid URL."
msgstr "Wprowadź poprawny adres URL."

#: fields.py:638
msgid "\"{value}\" is not a valid UUID."
msgstr "\"{value}\" nie jest poprawnym UUID."

#: fields.py:657
msgid "A valid integer is required."
msgstr "Wymagana poprawna liczba całkowita."

#: fields.py:658 fields.py:692 fields.py:725
msgid "Ensure this value is less than or equal to {max_value}."
msgstr "Upewnij się, że ta wartość jest mniejsza lub równa {max_value}."

#: fields.py:659 fields.py:693 fields.py:726
msgid "Ensure this value is greater than or equal to {min_value}."
msgstr "Upewnij się, że ta wartość jest większa lub równa {min_value}."

#: fields.py:660 fields.py:694 fields.py:730
msgid "String value too large."
msgstr "Za długi ciąg znaków."

#: fields.py:691 fields.py:724
msgid "A valid number is required."
msgstr "Wymagana poprawna liczba."

#: fields.py:727
msgid "Ensure that there are no more than {max_digits} digits in total."
msgstr "Upewnij się, że liczba ma nie więcej niż {max_digits} cyfr."

#: fields.py:728
msgid ""
"Ensure that there are no more than {max_decimal_places} decimal places."
msgstr "Upewnij się, że liczba ma nie więcej niż {max_decimal_places} cyfr dziesiętnych."

#: fields.py:729
msgid ""
"Ensure that there are no more than {max_whole_digits} digits before the "
"decimal point."
msgstr "Upewnij się, że liczba ma nie więcej niż {max_whole_digits} cyfr całkowitych."

#: fields.py:813
msgid "Datetime has wrong format. Use one of these formats instead: {format}."
msgstr "Wartość daty z czasem ma zły format. Użyj jednego z dostępnych formatów: {format}."

#: fields.py:814
msgid "Expected a datetime but got a date."
msgstr "Oczekiwano datę z czasem, otrzymano tylko datę."

#: fields.py:878
msgid "Date has wrong format. Use one of these formats instead: {format}."
msgstr "Data ma zły format. Użyj jednego z tych formatów: {format}."

#: fields.py:879
msgid "Expected a date but got a datetime."
msgstr "Oczekiwano daty a otrzymano datę z czasem."

#: fields.py:936
msgid "Time has wrong format. Use one of these formats instead: {format}."
msgstr "Błędny format czasu. Użyj jednego z dostępnych formatów: {format}"

#: fields.py:992 fields.py:1036
msgid "\"{input}\" is not a valid choice."
msgstr "\"{input}\" nie jest poprawnym wyborem."

#: fields.py:1037 fields.py:1151 serializers.py:482
msgid "Expected a list of items but got type \"{input_type}\"."
msgstr "Oczekiwano listy elementów, a otrzymano dane typu \"{input_type}\"."

#: fields.py:1067
msgid "No file was submitted."
msgstr "Nie przesłano pliku."

#: fields.py:1068
msgid ""
"The submitted data was not a file. Check the encoding type on the form."
msgstr "Przesłane dane nie były plikiem. Sprawdź typ kodowania formatki."

#: fields.py:1069
msgid "No filename could be determined."
msgstr "Nie można określić nazwy pliku."

#: fields.py:1070
msgid "The submitted file is empty."
msgstr "Przesłany plik jest pusty."

#: fields.py:1071
msgid ""
"Ensure this filename has at most {max_length} characters (it has {length})."
msgstr "Upewnij się, że nazwa pliku ma długość co najwyżej {max_length} znaków (aktualnie ma {length})."

#: fields.py:1113
msgid ""
"Upload a valid image. The file you uploaded was either not an image or a "
"corrupted image."
msgstr "Prześlij poprawny plik graficzny. Przesłany plik albo nie jest grafiką lub jest uszkodzony."

#: fields.py:1188
msgid "Expected a dictionary of items but got type \"{input_type}\"."
msgstr "Oczekiwano słownika, ale otrzymano  \"{input_type}\"."

#: pagination.py:221
msgid "Invalid page \"{page_number}\": {message}."
msgstr "Niepoprawna strona \"{page_number}\": {message}."

#: pagination.py:442
msgid "Invalid cursor"
msgstr "Niepoprawny wskaźnik"

#: relations.py:133
msgid "Invalid pk \"{pk_value}\" - object does not exist."
msgstr "Błędny klucz główny \"{pk_value}\" - obiekt nie istnieje."

#: relations.py:134
msgid "Incorrect type. Expected pk value, received {data_type}."
msgstr "Błędny typ danych. Oczekiwano wartość klucza głównego, otrzymano {data_type}."

#: relations.py:157
msgid "Invalid hyperlink - No URL match."
msgstr "Błędny hyperlink - nie znaleziono pasującego adresu URL."

#: relations.py:158
msgid "Invalid hyperlink - Incorrect URL match."
msgstr "Błędny hyperlink - błędne dopasowanie adresu URL."

#: relations.py:159
msgid "Invalid hyperlink - Object does not exist."
msgstr "Błędny hyperlink - obiekt nie istnieje."

#: relations.py:160
msgid "Incorrect type. Expected URL string, received {data_type}."
msgstr "Błędny typ danych. Oczekiwano adresu URL, otrzymano {data_type}"

#: relations.py:295
msgid "Object with {slug_name}={value} does not exist."
msgstr "Obiekt z polem {slug_name}={value} nie istnieje"

#: relations.py:296
msgid "Invalid value."
msgstr "Niepoprawna wartość."

#: serializers.py:299
msgid "Invalid data. Expected a dictionary, but got {datatype}."
msgstr "Niepoprawne dane. Oczekiwano słownika, otrzymano  {datatype}."

#: validators.py:22
msgid "This field must be unique."
msgstr "Wartość dla tego pola musi być unikalna."

#: validators.py:76
msgid "The fields {field_names} must make a unique set."
msgstr "Pola {field_names} muszą tworzyć unikalny zestaw."

#: validators.py:219
msgid "This field must be unique for the \"{date_field}\" date."
msgstr "To pole musi mieć unikalną wartość dla jednej daty z pola \"{date_field}\"."

#: validators.py:234
msgid "This field must be unique for the \"{date_field}\" month."
msgstr "To pole musi mieć unikalną wartość dla konkretnego miesiąca z pola \"{date_field}\"."

#: validators.py:247
msgid "This field must be unique for the \"{date_field}\" year."
msgstr "To pole musi mieć unikalną wartość dla konkretnego roku z pola \"{date_field}\"."

#: versioning.py:39
msgid "Invalid version in \"Accept\" header."
msgstr "Błędna wersja w nagłówku \"Accept\"."

#: versioning.py:70 versioning.py:112
msgid "Invalid version in URL path."
msgstr "Błędna wersja w ścieżce URL."

#: versioning.py:138
msgid "Invalid version in hostname."
msgstr "Błędna wersja w nazwie hosta."

#: versioning.py:160
msgid "Invalid version in query parameter."
msgstr "Błędna wersja w parametrach zapytania."

#: authtoken/serializers.py:20
msgid "User account is disabled."
msgstr "Konto użytkownika jest nieaktywne."

#: authtoken/serializers.py:23
msgid "Unable to log in with provided credentials."
msgstr "Podane dane uwierzytelniające nie pozwalają na zalogowanie."

#: authtoken/serializers.py:26
msgid "Must include \"username\" and \"password\"."
msgstr "Musi zawierać \"username\" i \"password\"."
